### ISSUE TITLE
Lost all Backups full and incremential ?

### ISSUE THREAD

**Issue by gueniewie** [nextcloud/backup#131]
I have installed Backup 1.0.3 NC 23
I have yesterday three backups 2 full one incremental but today i have lost all my backup on the UI website ?
This is the same Probleme on two serveres.

ON the path
/srv/ncdata/appdata...... 
I found my backups ?? but why i lost the all my backup

**Comment by gueniewie** [comment]
Hello all,
also with this 
sudo -u apache php /var/www/html/nextcloud/occ backup:point:list

I can't found my Backups ?

**Comment by ArtificialOwl** [comment]
You will need to be more precise about your setup, also do you have files in `data/appdata_???/backup/`

**Comment by gueniewie** [comment]
Yes on my local Drive I have the Files on my home server I have before 2 x full backup 1 x diferential today i have this ?
ncdata/appdata_???/backup/
drwxr-xr-x. 6 apache apache 143 20. Dez 16:54 20211220155054-differential-qpSS5rPiy7pXinO
drwxr-xr-x. 7 apache apache 155 23. Dez 16:50 20211223154648-differential-1uuyMn2JVmW9pnD
drwxr-xr-x. 7 apache apache 155 26. Dez 17:55 20211226165225-differential-xpHF1r58qSYh77D

on the second server from my friend i Have
ncdata/appdata_???/backup/
drwxr-xr-x. 2 apache apache   40 19. Dez 15:55 20211219145447-full-PyrmsMa3cbAawKa
drwxr-xr-x. 7 apache apache 4096 22. Dez 18:39 20211222171845-differential-51OPtHDHfhvOY4Q
drwxr-xr-x. 7 apache apache 4096 25. Dez 18:50 20211225173032-differential-28QY88XIR7ypAXm
drwxr-xr-x. 7 apache apache 4096 28. Dez 18:53 20211228173353-differential-fUdbte1pOG6TwBD

I have no listen in the Web UI and with 
sudo -u apache php /var/www/html/nextcloud/occ backup:point:list
- retreiving data from local

I have configured a external Files on this servere

What can I do to Help and to be more precise
[nextcloud.zip](https://github.com/nextcloud/backup/files/7789915/nextcloud.zip)

**Comment by ArtificialOwl** [comment]
try:

`./occ files:scan-app-data`
`./occ backup:point:scan`

then `./occ backup:point:list`

**Comment by gueniewie** [comment]
Hello,
I lost again my backups in the UI and ./occ backup:point:list
on the two servers nearly to the same time ???

My full-backups are now DELETED only the differential Backups exists ?
Is there a command to clear the Database (Mariadb) and start again from beginn ?

After your commands i have this in the ./occ backup:point:list but nothing in the UI

- retreiving data from local
 > found RestoringPoint 20220101192659-differential-9yiOpCdOmvZwn8M
 > found RestoringPoint 20220104230230-differential-UBLHL42OPc8X6Jm
 > found RestoringPoint 20220107231344-differential-5jEygwj9iyFFvLD
+----------------------------------------------+---------------------+-----------+----------+-------------------------------------+---------+-----------------------------+----------+----------------+--+
| Restoring Point                              | Date                | NC        | Database | Parent                              | Comment | Status                      | Instance | Health         |  |
+----------------------------------------------+---------------------+-----------+----------+--------------------------------+----------------------------------------------+---------------------+-----------+----------+-------------------------------------+---------+-----------------------------+----------+----------------+--+
| Restoring Point                              | Date                | NC        | Database | Parent                              | Comment | Status                      | Instance | Health         |  |
+----------------------------------------------+---------------------+-----------+----------+-------------------------------------+---------+-----------------------------+----------+----------------+--+
+----------------------------------------------+---------------------+-----------+----------+-------------------------------------+---------+-----------------------------+----------+----------------+--+
| Restoring Point                              | Date                | NC        | Database | Parent                              | Comment | Status                      | Instance | Health         |  |
+----------------------------------------------+---------------------+-----------+----------+-------------------------------------+---------+-----------------------------+----------+----------------+--+
|  20220101192659-differential-9yiOpCdOmvZwn8M | 2022-01-01 19:26:59 | 23.0.0.10 | mysql    | 20211229193029-full-fvc1KwzsBLtssOU |         | packed,compressed,encrypted | local    | without parent |  |
|  20220104230230-differential-UBLHL42OPc8X6Jm | 2022-01-04 23:02:30 | 23.0.0.10 | mysql    | 20211229193029-full-fvc1KwzsBLtssOU |         | packed,compressed,encrypted | local    | without parent |  |
|  20220107231344-differential-5jEygwj9iyFFvLD | 2022-01-07 23:13:44 | 23.0.0.10 | mysql    | 20211229193029-full-fvc1KwzsBLtssOU |         | not packed                  | local    | without parent |  |
+----------------------------------------------+---------------------+-----------+----------+--------------------------------

**Comment by ArtificialOwl** [comment]
Do you see the file in your app_data ?

to fully clean the app's settings:

`./occ backup:reset --uninstall`
`./occ app:enable backup`

**Comment by gueniewie** [comment]
I have only the differential files the full Backup is deleted ?

drwxr-xr-x. 6 apache apache 4096  2. Jän 20:59 20220102195809-differential-GCpGoiAL6xmOsOD
drwxr-xr-x. 7 apache apache 4096  5. Jän 21:08 20220105200841-differential-g5uRcd8gvglZGSQ
drwxr-xr-x. 7 apache apache 4096  9. Jän 00:30 20220108232912-differential-6SdHE6sO1T8zf5z

A full clean and resart backup, have after 4 -5 Days the same effect ;-( on tow systems NC23 backup 1.0.4

**Comment by Kuphi** [comment]
> to fully clean the app's settings:
> 
> `./occ backup:reset --uninstall` `./occ app:enable backup`

Please mention this in the readme. Currently there are still many issues which can be quickly fixed by a full reset only.
Took me quite some time until I finally found this.

Keep up the good work. Looking forward to a fully functional backup + restore app for Nextcloud in the future. Really a good app.
